### ISSUE TITLE
Clean up the settings tabs

### ISSUE THREAD

**Issue by willjasen** [willjasen/tephramesh#3]
- Topology should be called Health and keep the left color status dot
 - Graph should be called Topology

Tab ordering should be:

- Health
- Topology
- Metrics
- Instances
- Vault
- Config
- Signing

<img width="638" height="517" alt="Image" src="https://github.com/user-attachments/assets/283daba0-d3c3-4f9c-bc45-f1118ac752fb" />
